README.md

```diff
@@ -61,13 +109,14 @@ limitations under the License.
 ```
 
 ## Credits
-Maintained and sponsored by [Infinum](http://www.infinum.com).
 
-<p align="center">
-  <a href='https://infinum.com'>
+Maintained and sponsored by [Infinum](https://infinum.com).
+
+<div align="center">
```

Comment:
Uhm, when I preview it doesn't actually look centered 🤔 I think you have to use <p> tag like previously done if you want it centered.

Reply:
🤔 
Where are you doing preview?
To me looks ok here in github when I open readme as file. 
<img width="300" alt="image" src="https://github.com/infinum/android-library-template/assets/20267231/f4d91158-9e2c-4156-8d51-4b11136630a6">

[Also, on open source template page as well](https://github.com/infinum/open-source-mobile-template)

Reply:
I was looking at the preview in the diff, maybe it's messed up in the diff specifically. No worries then 👍